src/App.js

```diff
@@ -14,6 +16,10 @@ const App = () => {
         <Route path='/' element={<LandingPage />} />
         <Route path='/capture' element={<CameraCapture />} />
         <Route path='/leaderboard' element={<Leaderboard />} />
+        <Route path='/recycle' element={<CategoryDescription type={categoryData.recycle}/>} />
```

Comment:
all different routes? not on one page for clarity?

Reply:
Good point. I've changed it to a single page and will feed in the type based on the item using the photo's metadata.